llvm/test/Transforms/InstCombine/debuginfo-invert.ll

```diff
@@ -0,0 +1,70 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes=instcombine -S %s -o - | FileCheck %s
+
```

Comment:
Please can you add a  comment describing what's being tested? IMO this is especially important for debug info tests using update_test_checks.py.

I'm not a huge fan of using `update_test_checks.py` for debug info tests like this (they become difficult to understand, and increase the chance the test is erroneously updated with debug info errors in the future), but I don't think we have a concrete "policy" on it (people may disagree with my opinion on it...), and this test is simple enough, so I won't push back on it.

Reply:
Done